docs/examples/buttons/WarningButton/README.md

```diff
@@ -1 +1 @@
-undefined
\ No newline at end of file
+A form component that should be used to indicate the primary action to be taken on a particular page. Used be used when the primary action is one that you want to warn the user could have negative actions (such as removing an entity from your application). There should be only one WarningButton per page.
```

Comment:
Should be used when the primary action is one that entails negative actions (such as...)